### PR TITLE
Fix basal area methods

### PR DESCRIPTION
**Pull request by junrongYu**
Fixes: #34

### CODE REVIEW

**Comment by edmondchuc** [comment]
Once this is ready for review. Make an initial `0.1.0` release and see if the `deploy_to_graphdb.yml` workflow works correctly.

**Comment by junrongYu** [comment]
The query in `linkeddata-site` is not updated, so `skos:member` is kept to show members (`skos:memberList` cannot be seen).

**Comment by edmondchuc** [comment]
Let's wait until the viewer has been updated to support `skos:OrderedCollection` and `skos:memberList`.

**Comment by junrongYu** [comment]
Can we merge it first so that `basal area` can be used to test the new functionality?

**Comment by edmondchuc** [comment]
I see, you want to merge it to see how it is rendered for now and later we remove the `skos:member` statements.

**Comment by edmondchuc** [comment]
Yeah sure, we just need to remember to remove them later.

**Comment by junrongYu** [comment]
> I see, you want to merge it to see how it is rendered for now and later we remove the `skos:member` statements.

Yeah, I've tested it locally, `skos:OrderedCollection`, `tern:MethodCollection`, and `tern:Method` are presented correctly, `skos:memberList` cannot be fetched because the functionality is not implemented yet. When it's done, we can use `basal area` to test whether the submodules are presented in correct order (`full` - > `lite` - > `basal wedge`).

**Comment by junrongYu** [comment]
> Yeah sure, we just need to remember to remove them later.

Yes, surely we'll review it again. :)

**Review by edmondchuc — APPROVED** [review]

File: load.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from rdflib import Graph
 from requests.exceptions import HTTPError
 
-from dawe_nrm.graphdb import repository
```
[the remark this answers is not in the document]
Comment (junrongYu) [reply load.py]: Hi @edmondchuc , `src` is deleted.

**Pull request merged**